tasks/agent5-linux.yml

```diff
@@ -26,6 +26,21 @@
     enabled: no
   when: not datadog_skip_running_check and not datadog_enabled
 
+- name: Register all checks directories present in datadog
+  find:
+    paths: /etc/datadog-agent/conf.d/
```

Comment:
With agent 5 it's `/etc/dd-agent/` not `/etc/datadog-agent/`.

Reply:
Thx, I just fixed it